5.0/ubuntu/16.04/slim/Dockerfile

```diff
@@ -3,7 +3,7 @@ LABEL maintainer="Swift Infrastructure <swift-infrastructure@swift.org>"
 LABEL Description="Docker Container for the Swift programming language"
 
 RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && apt-get -q update && \
-    apt-get -q install -y \
+    apt-get --no-install-recommends install -y apt-utils ca-certificates \
```

Comment:
Please put the `apt-utils` and `ca-certificates` on their own line, to match the rest of the packages.